src/componenet/calculator.js

```diff
@@ -1,64 +1,80 @@
+import React, { useState } from 'react';
 import '../css/calculator.css';
+import calculate from '../logic/calculate';
 
 function MyComponent() {
+  const [preScreen, setPreScreen] = useState('');
+  const [curScreen, setCurScreen] = useState('0');
+  const [operation, setOperation] = useState(null);
+
+  const handleButtonClick = (buttonName) => {
+    const result = calculate({ total: preScreen, next: curScreen, operation }, buttonName);
+    setPreScreen(result.total || '');
+    setCurScreen(result.next || '0');
+    setOperation(result.operation || null);
+  };
+
   return (
     <div className="cal-container">
-      <div className="cal-screen">0</div>
-      <button className="cal-batn" type="button">
+      <div className="cal-screen">
+        <div className="pre-screen">{preScreen}</div>
+        <div className="cur-screen">{curScreen}</div>
```

Comment:
- [ ] [optional] @najibullahjafari Great work on your app, however, I would recommend a way to combine both outputs for both preScreen and curScreen, this will make the user experience better, currently, it creates a little confusion about whether the calculator is working correctly.
![image](https://github.com/najibullahjafari/math-magicians/assets/61408860/db518fff-efaf-43eb-885d-62b8065d9e0b)

Reply:
I think it looks like a professional calculator then a simple calculator.